pyiron/atomistics/master/elastic.py

```diff
@@ -0,0 +1,385 @@
+            strain=strain,
+            energy=energy,
+            volume=volume,
+            rotations=rotations,
+            max_polynomial_order=max_polynomial_order,
+            fit_first_order=fit_first_order,
+        )
+    else:
+        raise ValueError('Provide either strain and stress, or strain, energy '
+                         + 'and volume of same length.')
+    if return_score:
+        return np.array(coeff)[:,:6], score
+    else:
+        return np.array(coeff)[:,:6]
+
+def get_strain(max_strain=0.05, n_set=10, polynomial_order=3, additional_points=0, normalize=False):
+    strain_lst = np.random.random((n_set, 3, 3))-0.5
+    strain_lst += np.einsum('nij->nji', strain_lst)
+    if normalize:
+        strain_lst = np.einsum('nij,n->nij', strain_lst, 1/np.linalg.norm(strain_lst.reshape(-1, 9), axis=-1))
+    strain_lst *= max_strain
+    m = np.linspace(-1, 1, int(2*polynomial_order+2*additional_points-1))
+    strain_lst = np.einsum('k,nij->nkij', m, strain_lst).reshape(-1, 9)
+    return np.unique(strain_lst, axis=0).reshape(-1, 3, 3)
+
+class ElasticJobGenerator(JobGenerator):
+    @property
+    def parameter_list(self):
+        parameter_lst = []
+        for ii, epsilon in enumerate(self._job.input['strain_matrices']):
+            basis = self._job.ref_job.structure.copy()
+            basis.apply_strain(np.array(epsilon))
+            parameter_lst.append([ii, basis])
+        return parameter_lst
+
+    def job_name(self, parameter):
+        return "{}_{}".format(self._job.job_name, parameter[0]).replace(".", "_")
+
+    @staticmethod
+    def modify_job(job, parameter):
+        job.structure = parameter[1]
+        return job
+
+class ElasticTensor(AtomisticParallelMaster):
+    """
+    Class to calculate the elastic tensor and isotropic elastic constants
+
+    Example:
+
+    >>> job = pr.create_job('SomeAtomisticJob', 'atomistic')
+    >>> job.structure = pr.create_structure('Fe', 'bcc', 2.83)
+    >>> elastic = job.create_job('ElasticTensor', 'elastic')
+    >>> elastic.run()
+
+    Input parameters:
+
+        min_num_measurements (int): Minimum number of measurements/simulations to be launched
+        min_num_points (int): Minimum number of data points to fit data (number of measurements
+            times number of symmetry operations)
+        strain_matrices (numpy.ndarray): Strain tensors to be applied on simulation boxes (optional)
+        rotations (numpy.ndarray): Rotation matrices for box symmetry
+        use_symmetry (bool): Whether or not exploit box symmetry (ignored if `rotations` already specified)
+        use_elements (bool): Whether or not respect chemical elements for box symmetry (ignored if `rotations`
+            already specified)
+
+    The default input parameters might not be chosen adequately. If you have a large
+    computation power, increase `min_num_measurements`. At the same time, make
+    sure to choose an orientation which maximizes the number symmetry operations.
```

Comment:
I really like this added context! I would go even further and quickly explain *why* not supporting pressure is a problem, i.e. something like "--without pressure the constants must be fit to total cell energies, which contain only a fraction of the data that the pressure matrix does. This lack can be compensated by sampling more rotations." (or  however you want to word it.)